test/conftest.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; -*-
```

Comment:
This is unnecessary, we don't care about Python 2 anymore.

Reply:
This is not just about Python 2... though I would not fight too hard on this (I am old, I still work with Emacs)

Reply:
*works in Vim*